yaksh/documentation/moderator_docs/creating_lessons_modules.rst

```diff
@@ -0,0 +1,90 @@
+===================
+Lessons and Modules
+===================
+
+Courses can have lessons and quizzes encapsulated using a module.
+
+	* **What is a lesson?**
+		A lesson can be any markdown text or an embedded video of a particular topic.
+
+	* **What is a module?**
+		A module encapsulates lessons and quizzes. A module can have its own description 
```

Comment:
A Module is a collection of lessons and courses clubbed together by similar idea/content.